nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/adapter/RestAdapter.java

```diff
@@ -178,8 +178,25 @@ public void service(Request req, Response res) {
                     adapter = exposeContext();
                     RestLogging.restLogger.log(Level.INFO, RestLogging.REST_INTERFACE_INITIALIZED, context);
                 }
-                //delegate to adapter managed by Jersey.
-                adapter.service(req, res);
+
+                ClassLoader originalContextClassLoader = Thread.currentThread().getContextClassLoader();
+                boolean monitoring = false;
```

Comment:
Just a small OCD change, but could this line be simplified?
```suggestion
                boolean monitoring = context.equals("/monitoring");
```